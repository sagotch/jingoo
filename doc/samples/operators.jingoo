1 + 1 = {{ 1 + 1 }}
1 - 1 = {{ 1 - 1 }}
2 * 3 = {{ 2 * 3 }}
5 / 2 = {{ 5 / 2 }}
2 ** 8 = {{ 2 ** 8 }}
8 % 3 = {{ 8 % 3 }}
!true = {{ !true }}
not true = {{ not true }}
