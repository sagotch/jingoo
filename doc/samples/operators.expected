1 + 1 = 2
1 - 1 = 0
2 * 3 = 6
5 / 2 = 2
2 ** 8 = 256.
8 % 3 = 2
!true = false
not true = false
